lib/components/cards/SmallVeggieCard.dart

```diff
@@ -23,29 +23,27 @@ class SmallVeggieCard extends StatelessWidget {
         );
       },
       child: Container(
-        decoration: $styles.containerStyles.rounded,
-        child: Padding(
-          padding: EdgeInsets.all($styles.padding.m),
-          child: Row(
-            crossAxisAlignment: CrossAxisAlignment.start,
-            children: [
-              VeggieImage(veggie: veggie, size: size),
-              SizedBox(width: $styles.padding.s),
-              Expanded(
-                child: Column(
-                  crossAxisAlignment: CrossAxisAlignment.start,
-                  children: [
-                    CardHeader(veggie: veggie, size: size),
-                    Gap($styles.padding.s),
```

Comment:
Spacing is applied throughout the app with Paddings, SizedBoxes, Containers, and here Gap. I tried to simplify to reduce all the options.